koji/collection-mash-split.py

```diff
@@ -403,7 +403,6 @@ def main():
             "rhel7": "el7",
             "rhel6": "el6",
             "rhel5": "el5",
-            "fedora29": "fc29",
```

Comment:
This also removes it from all previously released versions. I'm not sure that's correct.

Reply:
Ahh right, ok added a condition to add it for relevant releases.